src/ssort/_files.py

```diff
@@ -1,72 +1,33 @@
-from __future__ import annotations
+from pathlib import Path
 
-import os
-import pathlib
-from typing import Iterable
+__all__ = ["find_project_root"]
 
-import pathspec
 
-from ssort._utils import memoize
+def current_working_dir():
+    return Path(".").resolve()
 
-_EMPTY_PATH_SPEC = pathspec.PathSpec([])
 
+def find_project_root(patterns):
```

Comment:
I think we need to track the `.git` root and the `pyproject.toml` root separately.

We want `.gitigore` files outside the `pyproject.toml` root to be respected but inside the `.git` root, but we want `project.toml` files outside the `.git` root to be ignored.